### PR TITLE
Browser and font fixes

### PR DESCRIPTION
**Pull request by pwithnall**
Various fixes for misc failures in https://openqa.dev.endlessm.com/tests/overview?distri=eos&version=master&build=181113-002311&groupid=1.

I haven’t been able to test them because I can’t currently log in to OpenQA. Discussing that with Dan on Slack.

### CODE REVIEW

File: main.pm [hunk 1]
```diff
@@ -57,7 +57,9 @@ if (!get_var('START_AFTER_TEST') && !get_var('BOOTFROM') && !get_var('FBE_TEST')
     autotest::loadtest "tests/_enable_coredumps.pm";
 
     # Disable notification popups because they get in the way of a lot of needles.
+    # Similarly, disable the screensaver.
     autotest::loadtest "tests/_disable_desktop_notifications.pm";
+    autotest::loadtest "tests/_disable_screensaver.pm";
```
Comment (wjt) [inline main.pm]: Very disappointed that this is not implemented in terms of jiggling the mouse every few minutes.

**Comment by wjt** [comment]
LGTM, merge if tests pass

File: main.pm (hunk 1 above)
Comment (pwithnall) [reply main.pm]: 🤷

**Comment by pwithnall** [comment]
The LO Writer test seems to be showing a window with no toolbars, sidebar or status bar, which is a bit odd: https://openqa.dev.endlessm.com/tests/5654#step/desktop_libreoffice_writer/4. I don’t know if that’s deliberate. Still waiting for the other test results.

**Comment by pwithnall** [comment]
I’m tempted to defer that to a follow-up issue though.

**Pull request merged**